README.md

```diff
@@ -377,3 +377,8 @@ This CVE has been reported in systems.text.regularexpressions.dll which is used
 * Delete any logs collected thus far and make sure that all copies are deleted. 
 * If you cannot upgrade for any reason, please ensure all debugging is disabled
 * If you are concerned about a potential compromise, contact Snowflake Customer Support for assistance with invalidating all active sessions/tokens. 
+
+3. Global http connection settings - 
+	Snowflake has identified an issue where the connector is globally enforcing TLS 1.2 and certificate revocation checks with connectors up to v1.2.1.
```

Comment:
Can you change "connectors up to v1.2.1" to "the .NET Driver v1.2.1 and earlier versions"?